packages/create-sitecore-jss/src/templates/react/data/routes/styleguide/{{language}}.yml

```diff
@@ -0,0 +1,26 @@
+fields:
+  pageTitle: Styleguide | Sitecore JSS
+placeholders:
+  <%- helper.getAppPrefix(appPrefix, appName)%>Web-jss-main:
```

Comment:
remove Web- from this file's placeholders
Also check Nextjsstyleguide template and/or angular for the update version of this yaml file. It shouldn't have any references to Danish